libpod/reset.go

```diff
@@ -104,8 +106,12 @@ func (r *Runtime) Reset(ctx context.Context) error {
 	}
 
 	if rootless.IsRootless() {
-		configPath := filepath.Join(os.Getenv("HOME"), ".config/containers")
-		if err := os.RemoveAll(configPath); err != nil {
+		if storageConfPath, err := storage.DefaultConfigFile(true); err == nil {
```

Comment:
This same check should be done as root, to see if the /etc/containers/storage.conf file has been created.